src/redux/store.js

```diff
@@ -0,0 +1,18 @@
+import { applyMiddleware, createStore, compose } from 'redux';
+import createSagaMiddleware from 'redux-saga';
+import allReducers from './reducers';
+import saga from "./sagas/rootSaga";
```

Comment:
Remember to update this import once you decide where to put the `rootSaga` (*or `allSagas` to be consistent with how you have named `allReducers`*) 🙂